Dockerfile

```diff
@@ -15,4 +15,8 @@ RUN systemctl set-default multi-user.target
 
 COPY setup /sbin/
 
-CMD ["/sbin/init"]
+STOPSIGNAL SIGRTMIN+3
+VOLUME ["/sys/fs/cgroup"]
```

Comment:
To me the use of the `VOLUME` command suggests that containers started from this image *provide* a volume to be mounted into other containers, which is the opposite of what we need here. At least for me the Dockerfile would be clearer without it, so could you please remove this line?